README.md

```diff
@@ -55,234 +20,85 @@ After you're finished please remove all the comments and instructions!
 - [🤝 Contributing](#contributing)
 - [⭐️ Show your support](#support)
 - [🙏 Acknowledgements](#acknowledgements)
-- [❓ FAQ (OPTIONAL)](#faq)
 - [📝 License](#license)
 
 <!-- PROJECT DESCRIPTION -->
```

Comment:
- [ ] [OPTIONAL] I kindly suggest you add a project description to make the README file look professional.